Bottom: bd6929cce5362623eb111921724545fa0c7e4515
Top:    6811cbe6f200e50e4c9eeefb06bca2405089f6a7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-12 01:07:42 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 6c8fabb..dd299b9 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
